.drone.star

```diff
@@ -27,23 +50,54 @@ def step(alpinever,tags=[]):
         "pull": "always",
         "settings": {
           "run": vertest + "su-exec nobody apk --version",
+          "repo": repo,
```

Comment:
Overwriting the repo variable from the global scope seems kinda cludgy, could cause some weirdness if the local-scope variable ever doesn't work.

Reply:
It's probably well-defined, but I'll rename for clarity